### ISSUE TITLE
AdditionalUserInfo `isNewUser` always returns `false`

### ISSUE THREAD

**Issue by dimovskidamjan** [firebase/firebase-ios-sdk#2346]
Environment

  * Xcode version: 10.1
  * Firebase SDK version: 5.16.0
  * Firebase Component: Auth
  * Component version: 5.16.0

Problem

#### Steps to reproduce:

I've implemented "Authenticate with Firebase Using Email Link in iOS" as per the docs, and in the completion handler of the signIn method I'm trying to check the "isNewUser" parameter, but it's always false.

#### Relevant Code:

The `print` statement always prints `false` in the chunk below
```
guard let link = userActivity.webpageURL?.absoluteString else { return false }

        if Auth.auth().isSignIn(withEmailLink: link) {
            Auth.auth().signIn(withEmail: email,
                               link: link) { [unowned self] (user, error) in
                                if let error = error {
                                    print(error.localizedDescription)
                                } else {
                                    if let user = user, let additionalInfo = user.additionalUserInfo {
                                        print(additionalInfo.isNewUser)
                                    }
                                }
            }
        }
```

**Comment by renkelvin** [comment]
@dimovskidamjan Thanks for reporting! It seems a bug of the SDK. I'll let you know when the fix is available.

**Comment by renkelvin** [comment]
The fix is checked in and will be available in next version (5.3.1)
